README.md

```diff
@@ -1,24 +1,62 @@
-# README
+# ObituaryHub
```

Comment:
Created a framework for the README for us to fill-in and fill out later.

Reply:
Right on, thanks for setting this up. I'll add a card to the project board for creating a README for the microservice repo as well.

Reply:
README Queen